trunner/device.py

```diff
@@ -18,12 +18,14 @@
 
 class RunnerFactory:
     @staticmethod
-    def create(target):
+    def create(target=None, serial=None):
```

Comment:
target should not have default value, it is explicitly needed when calling this method

Reply:
Corrected, thanks